src/main/python/setup.py

```diff
@@ -6,8 +6,8 @@
 setuptools.setup(
     name='rlbot',
     packages=setuptools.find_packages(),
-    install_requires=['psutil', 'inputs'],
-    version='0.0.9',
+    install_requires=['psutil', 'inputs', 'PyQt5'],
```

Comment:
why does this not just read from requirements.txt

It is bad to have to change 2 files every time we add a requirement

Reply:
Yeah, good point. I've gotten rid of the duplication by making requirements.txt refer to this.